server/static/js/index/home_page_preloader.js

```diff
@@ -0,0 +1,31 @@
+"use strict";
+const DEVICE_PRELOADER = '#table_device_preloader';
+const USERS_PRELOADER = '#table_users_preloader';
+
+// The function is intended to execute the script when sending an AJAX request.
+function beforeLoadTableDevice () {
+    $(TABLE_DEVICES).css(DISPLAY_NONE);
+    $(DEVICE_PRELOADER).css(DISPLAY_FLEX);
+}
+
+// The function is designed to run the script when the AJAX request is completed (displays a preloader, smoothly displays the table data)
```

Comment:
This function simply hides the devices table loading spinner. Please, reflect that in comment.
And yes - make it a proper JS docstring too.